packages/teleport/src/services/database/types.ts

```diff
@@ -17,7 +17,6 @@ limitations under the License.
 export interface Database {
   name: string;
   desc: string;
-  protocol: string;
   type: string;
   uri: string;
```

Comment:
don't need the uri field anymore

Reply:
Thought we might need it later for connecting to the DB

Reply:
We can add it back in the next issue if need